examples/Learning_example/learning_example.ipynb

```diff
@@ -0,0 +1,2295 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    pos1 = [0, 10000000, 0]
</pre><p>Just computed pos, v for two sats in similar orbits as sentinel. Just use v1 for both sats if you want them to move in opposing directions.</p><p> </p><p>sat 1</p><p>r at ref. = [-6361974.5152151017, 52215.393130868499, -345258.19754751836]</p><p>v at ref. = [433.46690432313761, 1180.9649876192523, -7808.7670808152052]</p><p>sat 2</p><p>r at ref. = [6361974.5152151017, -52215.393130868615, 345258.19754751911]</p><p>v at ref. = [-433.46690432313858, -1180.9649876192523, 7808.7670808152052]</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aidotse/PASEOS/pull/107/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>